spec/line-top-index-spec.js

```diff
@@ -0,0 +1,189 @@
+/** @babel */
```

Comment:
Maybe the specs here could benefit from some property (randomized) tests, as we do for other data structures such as the marker-index?